posts/2023-09-29-meetup-at-the-rotterdam-office-|-devoxx-try-out:-maintaining-maven-at-scale-&-machine-learning-in-java.adoc

```diff
@@ -0,0 +1,129 @@
+
+**Automated Remediation**
+
+Tim delves into the concept of real automated remediation, a process that involves creating a code model and transforming it into an abstract syntax tree. The extensive library of recipes available for various projects is a testament to the efficiency of this approach, offering declarative YAML and imperative Java visitors as tools in the arsenal.
+
+**Contributing to Open Source**
+
+Contributors play a vital role in automating static analysis issues and migration engineering. Tim explains the differences between OpenRewrite (open source) and Moderne (private SaaS platform), shedding light on the impactful changes made through contributions. He doesn't shy away from discussing the backlog of Maven Jira projects and the persistence needed to address long-standing tickets.
+
+**Humor and Challenges**
+
+Amidst the technical intricacies, Tim shares a light-hearted moment from the talk, offering a humorous take on contributor agreements. It's a reminder that open source is not only about code but also about the camaraderie that makes the journey enjoyable.
+
+**Contributing to Apache Maven**
+
+Tim provides a demonstration of how to contribute to the Maven project using Moderne. He explains how recipes are applied, patterns are sought, and transformations are made. The option to write recipes in Java and the inclusion of unit tests add depth to the contribution process.
+
+**Getting Started in Open Source**
+
+Tim wraps up by offering valuable advice on how to embark on your open-source journey. It starts with finding a project that aligns with your skills, following project guidelines, and starting with small but meaningful contributions. Patience and persistence are the keys to becoming a valued contributor.
+
+**Conclusion**
+
+In conclusion, Tim te Beek's talk on maintaining Maven at scale offers a glimpse into the world of open-source contributions and skills development. It highlights the critical role open source plays in the software development ecosystem and the enduring spirit of giving back. As we navigate the intricate landscape of Maven and other related projects, the future of open source shines bright, offering endless possibilities and opportunities for growth.
+
+The second talk was about Machine Learning in Java for sorting Lego bricks. In this enlightening talk, we step into the realm of machine learning, guided by the expertise of Jago de Vreede, a seasoned full-stack software engineer. Jago's unbridled enthusiasm for pushing the boundaries of Java led him on an extraordinary journey – harnessing the power of machine learning to tackle a tangible problem: automating the identification of Lego pieces in a sprawling bin.
+
```

Comment:
I think you can remove this paragraph and just go straight into the next section on the second talk